src/sinks/azure_monitor_logs.rs

```diff
@@ -212,7 +211,7 @@ impl HttpEventEncoder<serde_json::Value> for AzureMonitorLogsEventEncoder {
         let mut entry = serde_json::json!(&log);
         let object_entry = entry.as_object_mut().unwrap();
         object_entry.insert(
-            timestamp_key.to_string(),
+            log_schema().timestamp_key().to_string(),
```

Comment:
I'm not understanding why we're still using the `log_schema()` here?

I'm also not sure what we're doing here at all with the removal, json encoding, inserting after encoding...

I'd expect the other place you've asked about `log_schema()` to need to be the same key...